### ISSUE TITLE
Feature: Add workspace separation icon

### ISSUE THREAD

**Issue by rwjack** [fthx/babar#52]
![image](https://user-images.githubusercontent.com/59068073/186517388-c04465ac-52e3-462f-93ce-7d6c66eb6ac4.png)
^ These numbers seem to fat

Would be nice if the user could just chose their own separator, or at least have the option to have '|' as a separator.

Disabling the workspace numbers just groups all the icons and makes workspace separation confusing

**Comment by m-ar-c** [comment]
I agree, even without removing the number, I would like a kind of "|" separator too. I think that when [things are clearly separated](https://github.com/fthx/babar/issues/56) #56 it is easier to pick the relevant information.

**Comment by fthx** [comment]
Hi,

I do not work anymore on BaBar (of course you can do some PR). I do maintain DashBar now (but that's just the GS dash in panel, no diff workspaces nor settings).

**Comment by rwjack** [comment]
@m-ar-c Yeah I think I like your recommendation a bit more.

I did some digging, and basically the second IF statement is the pseudocode line we would need:

![image](https://user-images.githubusercontent.com/59068073/195822541-0cbf4e62-508a-4f6c-bc5e-36f435470116.png)

Any thoughts on how to get the WS index of the active window?

**Comment by rwjack** [comment]
Update on this: I've made it work in quite a hacky and broken way, but at least it looks as I wanted it to.

Since I use the workspace matrix extension (2Cols 4Rows), having visual separators really helps, also only the current workspace icons are colored, so the user can easily see which workspace is the active one.

![image](https://user-images.githubusercontent.com/59068073/210179225-fac9f68d-9ab0-435b-8430-45ba94ef45a4.png)
